lib/scalyr/common/client.rb

```diff
@@ -141,18 +153,48 @@ def initialize(logger, add_events_uri, compression_type, compression_level,
     end
   end  # def initialize
 
-
+  # Convenience method to create a fresh quantile estimator
+  def get_new_latency_stats
+    return {
+      # The total number of HTTP connections successfully created.
```

Comment:
Just more of a FYI for anyone else checking this code - estimator will keep a maximum of 512 values / samples (https://github.com/matttproud/ruby_quantile_estimation/blob/master/lib/quantile/estimator.rb#L113) so we have an upper mound on memory use.

And 512 should be fined as long as we don't have too many estimator classes (since each class instance will have a buffer / window for 512 samples).